.github/workflows/ci.yml

```diff
@@ -97,6 +97,8 @@ jobs:
           - { online: sqlite,   offline: sqlite,   metadata: sqlite   }
           - { online: mysql,    offline: mysql,    metadata: mysql    }
           - { online: postgres, offline: postgres, metadata: postgres }
+          - { online: redis, offline: postgres, metadata: postgres }
+          - { online: redis, offline: postgres, metadata: postgres }
```

Comment:
hmm is this a typo? seems you added two same combinations

Reply:
good catch! fixed.